components/ProgressBars/index.js

```diff
@@ -0,0 +1,138 @@
+import {
+  CircularProgressbar,
+  CircularProgressbarWithChildren,
+  buildStyles,
+} from "react-circular-progressbar";
+import { ProgressbarContainer } from "./ProgressBar.styled";
+import { useState, useEffect } from "react";
+
+export default function ProgressBars({
+  todaysKcal,
+  todaysCarbs,
+  todaysProtein,
+  todaysFat,
+  goal,
+}) {
+  const [kcalProgress, setKcalProgress] = useState(todaysKcal);
+  const [carbsProgress, setCarbsProgress] = useState(todaysCarbs);
+  const [proteinProgress, setProteinProgress] = useState(todaysProtein);
+  const [fatProgress, setFatProgress] = useState(todaysFat);
+
+  useEffect(() => {
+    setKcalProgress(todaysKcal);
+    setCarbsProgress(todaysCarbs);
+    setProteinProgress(todaysProtein);
+    setFatProgress(todaysFat);
+  }, [todaysKcal, todaysCarbs, todaysProtein, todaysFat]);
+
+  return (
+    <ProgressbarContainer>
+      <CircularProgressbarWithChildren
+        value={kcalProgress}
+        maxValue={goal.kcal}
+        circleRatio={0.5}
+        strokeWidth={10}
+        styles={{
+          root: {},
+          path: {
+            stroke: "var(--color-fg-kcal)",
+
+            strokeLinecap: "round",
+            transition: "stroke-dashoffset 1s ease 0.3s",
+            transform: "rotate(-0.25turn)",
+            transformOrigin: "center center",
+          },
+
+          trail: {
+            stroke: "var(--color-bg-progressbars)",
+
+            strokeLinecap: "round",
+            transform: "rotate(-0.25turn)",
+            transformOrigin: "center center",
+          },
+        }}
+      >
+        <div style={{ width: "74%", height: "77%" }}>
+          <CircularProgressbarWithChildren
+            value={carbsProgress}
+            maxValue={goal.carbs}
+            circleRatio={0.5}
+            strokeWidth={12}
+            styles={{
+              root: {},
+              path: {
+                stroke: "var(--color-fg-carbs)",
+
+                strokeLinecap: "round",
+                transition: "stroke-dashoffset 1s ease 0.3s",
+                transform: "rotate(-0.25turn)",
+                transformOrigin: "center center",
+              },
+              trail: {
+                stroke: "var(--color-bg-progressbars)",
+
+                strokeLinecap: "round",
+                transform: "rotate(-0.25turn)",
+                transformOrigin: "center center",
+              },
+            }}
+          >
+            <div style={{ width: "68%", height: "72%" }}>
```

Comment:
Use styled components here to get rid of the inline styles. Of course, this would only work if you changed the before mentioned hard coded values into variables i.e. `const progressBarWidth = "74%"` and `const progressBarHeight = "77%"`. View this as a suggestion. 

```suggestion
   const Wrapper = styled.div`
  width: ${progressBarWidth};
  height: ${progressBarHeight};
`;
```